### DISCUSSION TITLE
User::factory()->create() throws error

### DISCUSSION THREAD

**Discussion by rockwellll** [laravel/framework#34240] [General] [answered]
- Laravel Version: 8
- PHP Version: 7.3.4
- Database Driver & Version: MySql

### Description:
so after migrating from Laravel 7.x to 8, everything works good, but i have this problem with factories throwing error

whenever i try to run `User::factory()->create() `

i get the following error 

![image](https://user-images.githubusercontent.com/38360603/92658464-7f762500-f2ab-11ea-84fd-d372a0ba9356.png)

and the UserFactory class is from the Laravel docs 
### Steps To Reproduce:
php artisan make:factory UserFactory
boot up tinker
User::factory()->create() 

Thanks

**Comment by martijnd** [comment, marked as answer]
Set your all factory classes namespace to `Database\Factories`
